### PR TITLE
Update YesSql

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

**Comment by sebastienros** [comment]
To be discussed. Current Sqlite 6.0.0 will fail on netcoreapp3.1, on some environments.

**Comment by deanmarcussen** [comment]
It might be good to do another release with 6.0 before dropping 3.1.

File: src/OrchardCore.Modules/OrchardCore.Workflows/Http/Scripting/HttpMethodsProvider.cs [hunk 1]
```diff
@@ -174,13 +173,13 @@ public HttpMethodsProvider(IHttpContextAccessor httpContextAccessor)
                                     throw new Exception("Invalid form data passed in the request.");
                                 }
                             }
-                            else if (HasJsonContentType(httpContextAccessor.HttpContext.Request))
```
Comment (agriffard) [inline src/OrchardCore.Modules/OrchardCore.Workflows/Http/Scripting/HttpMethodsProvider.cs:177]: Then, remove `HttpMethodsProvider.HasJsonContentType` ?

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Workflows/Http/Scripting/HttpMethodsProvider.cs:177]: I did, didn't I ?

**Comment by sebastienros** [comment]
I handled the issue with netcoreapp3.1 by using the previous sqlite version in this case.

**Pull request merged**